src/Api/ItemDataProviderInterface.php

```diff
@@ -22,8 +23,6 @@
 interface ItemDataProviderInterface
 {
     /**
-     * Retrieves an item.
```

Comment:
Removed comment as is not very helpful IMO: we already have this description in the interface doc and the method name is quite explicit already.